packages/analytics-core/src/config.ts

```diff
@@ -123,7 +123,7 @@ export class RequestMetadata implements IRequestMetadata {
   constructor() {
     this.sdk = {
       metrics: {
-        histogram: new HistogramOptions(),
+        histogram: {},
```

Comment:
Is this intentional?

Reply:
Yes. `new HistogramOptions(),` will set every filed to undefined while `{}` won't.